src/internal/utils/mdObjectHelper.ts

```diff
@@ -126,3 +126,25 @@ export function countItemsOnAxes(
         secondary_yaxis: secondaryMeasureCountInConfig,
     };
 }
+
+export function isAttributeItem(
```

Comment:
is not the same?
https://github.com/gooddata/gooddata-typings/blob/master/src/VisualizationObject.ts#L202

Reply:
Yes, it is. Fixed